ambry-frontend/src/main/java/com.github.ambry.frontend/FrontendMetrics.java

```diff
@@ -117,7 +117,8 @@
   // PostCallback
   public final Counter postCallbackProcessingError;
   public final Counter outboundIdConversionCallbackProcessingError;
-
+  // TTL too large error
```

Comment:
minor: Does this need a section of its own? Should either go with the other errors or go into "Other" - this comment on its own is not very useful